index.js

```diff
@@ -172,18 +208,20 @@ export async function registerSpace (email) {
   const spinner = ora('registering your space').start()
 
   try {
-    await client.registerSpace(email)
+    await client.registerSpace(accountEmail, { provider: opts.provider })
```

Comment:
Probably one for later but I feel like this should take the DID of the account you want to register the space with.

Reply:
yep I agree - I think I'm going to move email to an `--email` option which would let us add an `--account` option or something pretty seamlessly later on